rpc/core/mempool.go

```diff
@@ -255,27 +255,32 @@ func BroadcastTxCommit(tx types.Tx) (*ctypes.ResultBroadcastTxCommit, error) {
 //
 // ```json
 // {
-//   "error": "",
-//   "result": {
-//     "txs": [],
-//     "n_txs": "0"
-//   },
-//   "id": "",
-//   "jsonrpc": "2.0"
-// }
+//   "result" : {
+//       "txs" : [],
+//       "total_bytes" : "0",
+//       "count" : "0",
```

Comment:
I feel like this would be more straight forward as:

```
txs
n_txs
total_txs
total_bytes
```

Otherwise it's not really clear what `count` and `total` are referring to. This would also make it non breaking, which is always a bonus :)

Reply:
The benefit of count and total is they are the same across all the endpoints (which return an array)

Reply:
Where else do we do that? At least for this PR since we made it non breaking we should fix the comment here